src/utils/editor.ts

```diff
@@ -47,6 +47,10 @@ export const COMPONENTS: ComponentType[] = [
   'Text',
   'Textarea',
   'Tab',
+  'TabList',
```

Comment:
Can you add these to the test file please ? https://github.com/premieroctet/openchakra/blob/master/src/components/editor/ComponentPreview.test.tsx#L72